.travis.yml

```diff
@@ -38,7 +42,8 @@ matrix:
         - cd ../test-create-project-dir
         - echo 'DATABASE_URL=mysql://root:@127.0.0.1:3306/sulu_test' >> .env.local
       env:
-        - NPM_BUILD: true
+        - NPM_BUILD=true
+        - COMPOSER_VALIDATE_FLAGS="--strict --no-check-publish"
```

Comment:
Because `sulu/sulu-minimal` is getting published so we should atleast in one run validate if its valide composer.json for publishing. symfony/flex when create a project does remove some required publish fields from composer.json e.g. name and description. Thats why in the test env with creat project we need to add this flag